src/main.rs

```diff
@@ -20,44 +20,51 @@ fn main() {
 
     let filepath = &args[1];
 
-    javap(filepath);
+    let javap_result = javap(filepath, false);
```

Comment:
We should at some point add some cli arguments to do things like allow for the user to request the code to be printed. Ideally we could replicate the functionality of the `-v` flag from `javap`.

But this should be fine for this PR.

Reply:
I opened #6 for this.